submissions/description/broken_access_control/idor/Guidance.md

```diff
@@ -0,0 +1,23 @@
+# Guidance
+
+<!--
```

Comment:
These guidance sections need to fit in a small hint text block next to the description field. Or can be omitted entirely! We can provide a default guidance text block if this is not specified.

Reply:
Please see recent commit for condensed guidance.md update.
I'm open to omitting the file entirely and creating a standalone page for guidance on writing up findings.